src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -166,6 +166,15 @@ class ConfigurationHelper implements Serializable {
         return this
     }
 
+    ConfigurationHelper withPropertyInList(String key, List values){
```

Comment:
Basically this is not a list but a Set. The order inside the list is significant, but here the order is not significant.

Reply:
Done